identities/models.py

```diff
@@ -44,3 +44,11 @@ class Identity(models.Model):
 
     def __str__(self):
         return str(self.id)
+
+
+class OptOut(models.Model):
+    """An opt_out"""
+    identity = models.ForeignKey(Identity)
```

Comment:
the address type and address should be not null because they'll be the minimum data set used to lookup the identity (if it's not explicitly set)